packages/platform-core/src/__tests__/core.test.ts

```diff
@@ -20,23 +20,25 @@ import startPlugin from '../plugin'
 import model from '../__model__/model'
 import Builder from '../__model__/builder'
 import core from '../__model__'
-
+import {mergeIds} from "../__model__/utils";
 const DOC = 1 // see `plugin.ts`
 
 describe('core', () => {
   const platform = new Platform()
 
   it('should load model', async () => {
-    const builder = new Builder()
+    const tx = await startPlugin(platform)
+    const builder = new Builder(tx.getDb())
```

Comment:
why you're reverting to this? I get rid of this in favor of `platform.setMetadata(core.metadata.MetaModel, coreModel)`